lib/Gemfile

```diff
@@ -76,5 +77,9 @@ unless ENV['APPLIANCE']
     gem "rspec-fire", "~>1.3.0",  :require => false
     gem "timecop",    "~>0.5.3",  :require => false
     gem "xml-simple", "=1.0.12",  :require => false  # Used by test/xml/tc_xmlhash_methods.rb
+    gem "vcr",        "~>2.6",    :require => false  # Used by manageiq-foreman
+    gem "webmock",                :require => false  # Used by vcr / manageiq-foreman
   end
 end
+
+gem 'pry'
```

Comment:
This should be removed...it will end up affecting the build team.  Additionally, I don't want pry to become a dependency for ManageIQ.  That is what the Gemfile.dev.rb file is for.